app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::Base
+  include Pundit
```

Comment:
would it be possible to add these checks?
```
  after_action :verify_authorized, except: :index, unless: :devise_controller?
  after_action :verify_policy_scoped, only: :index
```
https://github.com/varvet/pundit#ensuring-policies-and-scopes-are-used

Reply:
Well I liked the intent on this, but i'm having problems because adding that involves ALL controllers in the base, so we should be excluding all controllers besides `user_controller`, and I don't really like it...
opinions?

Reply:
I strongly recommend adding them. Otherwise people will forget about pundit not call the `authorize` method. You can ignore those controllers if it doesn't make sense to authorize there. For example we have `after_action :verify_authorized, except: :index, unless: :devise_controller?`

Reply:
not sure if you can use `, unless: :devise_controller?` or something similar

Reply:
@fedeagripa what happens with the settings controller? do you need to exclude it too?

Reply:
I ended up excluding the after actions in the controller itself, as that endpoint does not even requiere auth I think there is no need to policies.
Do you think it worth to add a policy specific to return true?

Reply:
I'm ok with that, I just hope it doesn't set a bad precedent 🙈